doc/source/developing.rst

```diff
@@ -16,9 +16,8 @@ zest.releaser checkout, do this::
 Python versions
 ---------------
 
-The tests currently pass on python 2.6 and 2.7. Using
-2.6.2 specifically will give test failures. Travis continuous integration
-tests 2.6 and 2.7 for us automatically.
+The tests currently pass on python 2.7 and 3.4. Travis continuous
+integration tests 2.7 and 3.4 for us automatically.
```

Comment:
Not explicitly, just updating the docs to reflect statements made elsewhere
on the release of 14.0.
On 2 Jun 2015 7:33 am, "Marius Gedminas" <notifications@github.com> wrote:

> In doc/source/developing.rst
> <https://github.com/zestsoftware/zest.releaser/pull/97#discussion_r31495734>
> :
>
> > @@ -16,9 +16,8 @@ zest.releaser checkout, do this::
> >  Python versions
> >  ---------------
> >
> > -The tests currently pass on python 2.6 and 2.7. Using
> > -2.6.2 specifically will give test failures. Travis continuous integration
> > -tests 2.6 and 2.7 for us automatically.
> > +The tests currently pass on python 2.7 and 3.4. Travis continuous
> > +integration tests 2.7 and 3.4 for us automatically.
>
> Does this drop 2.6 compat? (I'm not complaining, just wondering why that
> isn't mentioned in the changelog.)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zestsoftware/zest.releaser/pull/97/files#r31495734>.
>